x-pack/plugins/ml/public/application/explorer/explorer_constants.ts

```diff
@@ -55,3 +56,7 @@ export const MAX_INFLUENCER_FIELD_NAMES = 50;
 export const VIEW_BY_JOB_LABEL = i18n.translate('xpack.ml.explorer.jobIdLabel', {
   defaultMessage: 'job ID',
 });
+
+export const QUERY_LANGUAGE_KUERY = 'kuery';
```

Comment:
Can we use the `SEARCH_QUERY_LANGUAGE` enum from `ml/common/constants/search.ts` rather than redefining these here?